app/shared_features/set_card/expanded/VideoButton.js

```diff
@@ -6,22 +6,42 @@ import {
     TouchableOpacity,
     TouchableHighlight,
     Image,
+    Alert,
     Platform
 } from 'react-native';
 import Icon from 'react-native-vector-icons/FontAwesome';
 import Video from 'react-native-video';
+import Permissions from 'react-native-permissions';
 
 class VideoButton extends Component {
```

Comment:
- [ ] Instead of checking on record and such, have the black screen itself showcase the error message in the middle of it, kinda like an overlay